reportssql/report_util_functions.sql

```diff
@@ -130,6 +130,30 @@ BEGIN
 END$$
 DELIMITER ;
 
+-- patientHasRegisteredWithinReportingPeriod
```

Comment:
@prashantsable1991 Can you please update this function so that it returns true/false instead of Yes/No?